app/components/Onboarding/NewWalletSeed/NewWalletSeed.scss

```diff
@@ -18,7 +18,6 @@
 
       &:nth-child(1) {
         width: 15%;
-        text-align: center;
```

Comment:
I removed this so the left row of seed words is aligned with the left edge of the warning label box. I see the reason for centering it though. I think either looks ok.